testing/ui-tests/ui-test-users-id-tasks-id.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
+
+    (async () => {
+      const browser = await puppeteer.launch({headless: true});
```

Comment:
good job on making it headless

Reply:
FYI puppeteer runs headless by default! While it is totally fine to write `headless: true`, if you omit it, this also works. 
https://developers.google.com/web/tools/puppeteer/get-started